hbase-protocol-shaded/src/main/protobuf/server/master/Master.proto

```diff
@@ -437,10 +437,13 @@ message IsCleanerChoreEnabledResponse {
 
 message SnapshotRequest {
   required SnapshotDescription snapshot = 1;
+  optional uint64 nonce_group = 2 [default = 0];
```

Comment:
Why we do not have this in the past?

Reply:
Have no idea. Maybe the previous snapshot framework is safe enough to avoid repeated execution. But I think we should have this for the snapshot procedure so I added here.